editor/plugins/asset_library_editor_plugin.cpp

```diff
@@ -1242,15 +1248,28 @@ void EditorAssetLibrary::_http_request_completed(int p_status, int p_code, const
 			library_vb->add_child(asset_bottom_page);
 
 			if (result.is_empty()) {
+				String support_list;
+				for (int i = 0; i < SUPPORT_MAX; i++) {
+					if (support->get_popup()->is_item_checked(i)) {
+						if (!support_list.is_empty()) {
+							support_list += ", ";
+						}
+						support_list += TTR(support_ttr_key[i]);
```

Comment:
This won't work to extract the translation strings, `TTR()` needs to be used with a literal.
You should use `TTRGET()` here, and `TTRC()` in the Array.

Reply:
Ahh, I see, thanks! 
Done. (All your comments)